vmdb/app/models/mixins/service_orchestration_mixin.rb

```diff
@@ -0,0 +1,26 @@
+module ServiceOrchestrationMixin
```

Comment:
OrchestrationMixin was pulled from ServiceTemplateOrchestration and now shared by ServiceTemplateOrchestration and ServiceOrchestration